src/ipips/ipip-0383.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "IPIP-0383: Compact Denylist Format"
+date: 2023-03-09
+ipip: proposal
+editors:
+  - name: Hector Sanjuan
+    github: hsanjuan
+    affiliation:
+        name: Protocol Labs
+        url: https://protocol.ai/
+relatedIssues:
+  - https://github.com/ipfs/specs/issues/298
+  - https://github.com/ipfs/specs/pull/299
+  - https://github.com/ipfs/specs/pull/340
+order: 383
+tags: ['ipips']
+---
+
+## Summary
+
+This IPIP introduces a line-based denylist format for content blocking on IPFS
+focused on simplicity, scalability and ease-of-use.
+
+A reference Go implementation of a denylist parser and Blocker component for
+the Kubo (go-ipfs) stack exists at https://github.com/ipfs-shipyard/nopfs.
+
+## Motivation
+
+IPFS implementations should support content moderation, particularly when it
+comes to deployments of publicly-accessible infrastructure like gateways.
+
+The first step in a larger strategy to enable decentralized content moderation
+in IPFS setups is to agree in a denylist format that different implementations
+can rely on and share.
+
+## Detailed design
+
+See :cite[compact-denylist-format].
```

Comment:
:information_source:  This reference will start working once we have the spec published at https://specs.ipfs.tech/compact-denylist-format